### PR TITLE
Use new charmhelpers

### PR DESCRIPTION
**Pull request by bac**

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/55/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/56/
Test PASSed.

File: Makefile [hunk 1]
```diff
@@ -98,6 +98,10 @@ releases:
 .PHONY: package
 package: clean releases
 
+.PHONY: sync
+sync: charm-helpers.yaml
```
Comment (jaycee) [inline Makefile]: Unless charm-helpers.yaml might not be there, there's no need for this to be a requirement for the target.

**Comment by jaycee** [comment]
:+1: looks good, @bac. Thanks!

File: hooks/config-changed [hunk 2]
```diff
@@ -19,10 +19,8 @@
 
 import sys
 
-from charmhelpers import (
-    get_config,
-    log,
-)
+from charmhelpers.core.hookenv import log
+from charmhelpers.contrib.charmsupport.volumes import get_config
```
Comment (frankban) [inline hooks/config-changed]: Alphabetical order. I know I know...

File: hooks/utils.py [hunk 3]
```diff
@@ -30,15 +30,13 @@
 import apt
 import tempita
 
-from charmhelpers import (
+from charmhelpers.contrib.charmsupport.volumes import get_config
```
Comment (frankban) [inline hooks/utils.py]: Ditto.

**Comment by frankban** [comment]
:+1: with a couple of trivial changes.

File: Makefile (hunk 1 above)
Comment (bac) [inline Makefile]: Good call.

File: hooks/utils.py (hunk 3 above)
Comment (bac) [inline hooks/utils.py]: I don't see an ordering problem in this file.

File: hooks/config-changed (hunk 2 above)
Comment (bac) [inline hooks/config-changed]: Done

File: hooks/utils.py (hunk 3 above)
Comment (frankban) [inline hooks/utils.py]: And you are right, nm.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/58/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/59/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/60/
Test PASSed.

**Comment by bac** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-charm-merge

**Pull request merged**